manifests/host.pp

```diff
@@ -4,7 +4,8 @@
   $ip,
   $mac,
   $options = {},
-  $comment=''
+  $comment='',
+  $ignored = false
```

Comment:
Nit: I think trailing comma's are a good thing because it keeps the diff smaller when you add another parameter.